### ISSUE TITLE
clangd version parsing fails on Ubuntu when using clangd installed by apt

### ISSUE THREAD

**Issue by scarf005** [clangd/vscode-clangd#384]
This issue is very similar to #167.
1. installed clangd-14 using `sudo apt install clangd-14`
2. configured clangd path to be `/usr/bin/clangd-14`.
3. `clangd: Check for language server update` gives me

```
Failed to check for clangd update: Error: Couldn't parse clangd --version output: Ubuntu clangd version 14.0.0-1ubuntu1
Features: linux+grpc
Platform: x86_64-pc-linux-gnu
```
**Logs**
```
> clangd-14 --version
Ubuntu clangd version 14.0.0-1ubuntu1
Features: linux+grpc
Platform: x86_64-pc-linux-gnu
> apt show clangd-14
Package: clangd-14
Architecture: amd64
Version: 1:14.0.0-1ubuntu1
Priority: optional
Section: universe/devel
Source: llvm-toolchain-14
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: LLVM Packaging Team <pkg-llvm-team@lists.alioth.debian.org>
```

**System information**
Clangd version (from the log, or `clangd --version`): `Ubuntu clangd version 14.0.0-1ubuntu1`
clangd extension version: 0.1.21
Operating system: Kubuntu 22.04 x64

**Comment by HighCommander4** [comment]
This is in fact the same issue as #167.

Unfortunately, it turns out that the fix for #167 has not yet appeared in a vscode-clangd release. The dependent package node-clangd which contains the fix needs a new version published on npm so that vscode-clangd can pick up the fix; filed https://github.com/clangd/node-clangd/issues/18 for this.

**Comment by sam-mccall** [comment]
Sorry about that. All the bits are now released, should be fixed in vscode-clangd 0.1.23
